src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -130,6 +130,13 @@ is the general extensibility point to add additional properties to any activity.
 The `Enrich` option is specific to this instrumentation, and is provided to
 get access to `HttpRequest` and `HttpResponse`.
 
+### RecordException
```

Comment:
Will do follow up PR to make all other instrumentation support this feature (wherever feasible).